gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ConfidentialityUtil.java

```diff
@@ -0,0 +1,42 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.hl7.fhir.dstu3.model.Coding;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.v3.CV;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+
+public final class ConfidentialityUtil {
+    private ConfidentialityUtil() { }
```

Comment:
I think it would be good to break a line after line 11 otherwise it is very difficult to read such a class.
Also, it would be good to have a line break between the constructor and the static variable.